src/modules/jcms_rest/src/Plugin/rest/resource/BlogArticleItemRestResource.php

```diff
@@ -64,8 +64,12 @@ public function get($id) {
         }
       }
 
-      if ($content = $this->processFieldContent($node->get('field_content'))) {
-        $response['content'] = $content;
+      if ($node->hasField('field_processed_json')) {
```

Comment:
I like the idea. I am not aware of anything particularly dynamic inside the JSON, so the inputs would change only when editing the item from the interface.